cmd/generate/generate.go

```diff
@@ -112,6 +113,7 @@ func main() {
 	pflag.StringVar(&output, "output", filepath.Join(wd, "openshift"), "Output directory")
 	pflag.StringVar(&projectFilePath, "project-file", filepath.Join(wd, "openshift", "project.yaml"), "Project metadata file path")
 	pflag.StringVar(&dockerfileImageBuilderFmt, "dockerfile-image-builder-fmt", builderImageFmt, "Dockerfile image builder format")
+	pflag.StringVar(&mustGatherBuilderImageFmt, "must-gather-builder-image-fmt", mustGatherBaseImageFmt, "Must gather Dockerfile image builder format")
```

Comment:
Not sure if we need a new param for this, or if we could simply default to `quay.io/openshift/origin-must-gather:%s`